cabal.project.coverage

```diff
@@ -4,7 +4,7 @@ packages:
 source-repository-package
   type: git
   location: https://github.com/composewell/hpc-coveralls
-  tag: d05f0c68aabec6120f8a96a9b5181b21a8f364df
+  tag: 93f3dcd473b3f92bbb40b9d3db9c83f4bf085cff
```

Comment:
Update the commit to the latest version/tag instead of using the master?

Reply:
Nvm, We are not releasing a new version and even if we are `cabal.project.coverage` is not a part of sdist.

Reply:
This commit is not on master, it's on a different branch. cabal occasionally runs into weird problems when I use branch or tag name instead of the commit id.